spec/models/miq_provision_workflow_spec.rb

```diff
@@ -60,13 +67,53 @@
           MiqPassword.decrypt(request.options[:root_password]).should == password_input
         end
 
+        it "should set values when extra '|' are passed in" do
+          request = ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow.from_ws(
+            "1.1", admin, "name=template", "vm_name=spec_test",
+            nil, nil, {'abc' => 'tr|ue', 'blah' => 'nah'}, nil, nil)
```

Comment:
For the tests I was thinking we would pass in only one argument type (string or hash, not mixed).  I suggest that we convert most to use the hash format and have one that still uses the string format until that format support is removed.

Also, passing the string format generates "DEPRECATION WARNING:" messages in the test logs.  This will be reduced by changing the majority of the tests to use hash format and the test(s) using the string format should be wrapped with 
```
Vmdb::Deprecation.silence do
  ...
end
```